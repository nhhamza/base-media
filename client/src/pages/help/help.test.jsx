import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';

import configureStore from 'redux-mock-store';
import HelpPage from './help.component';

describe('HelpPage component', () => {
  let wrapper;
  const mockStore = configureStore([]);

  const store = mockStore({
    callCenter: {
      callCenterList: [{ id: '1', name: 'operator1', number: '1111131313' }],
    },
    header: { title: 'Help' },
  });

  beforeEach(() => {
    wrapper = mount(
      <Provider store={store}>
        <BrowserRouter>
          <HelpPage />
        </BrowserRouter>
      </Provider>
    );
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
