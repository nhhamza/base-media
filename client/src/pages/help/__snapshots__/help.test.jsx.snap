// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpPage component should render correctly 1`] = `ReactWrapper {}`;
